### ISSUE TITLE
How to train your designer?

### ISSUE THREAD

**Issue by papirfecnike** [BarcelonaJS/talks#6]
**NOTE:** The speaker is from Oslo and needs one month's notice.

### Talk Description

How can one effectively engage with a designer in a collaborative effort? What strategies and methods should be employed when communicating with a designer, and how can one articulate requests and inquiries? In what ways can a designer deliver results that align with your requirements for practical use? Is there room for refining communication skills or expressing thoughts in alternative ways? Are there approaches to integrate a designer into your workflows, fostering an environment where they naturally align their processes with yours for future collaborations?

### Speaker Name

Dora Makszy

### Speaker Position

Product design lead

### Speaker LinkedIn

https://www.linkedin.com/in/makszydora/

### Speaker Twitter

heydorka

### Number of Talks Given Before

3-5

### Speaker Company Name

Futurehome

### Speaker Company Website

https://futurehome.no/

### Speaker Company LinkedIn

https://www.linkedin.com/company/future-home/

### Speaker Company Twitter

n/a

### Code of Conduct and Terms and Conditions

- [X] I accept the Code of Conduct and Terms and Conditions (Being Recorded)

**Comment by onedesert** [comment]
Thanks for your submission.

**Comment by onedesert** [comment]
Hello! Could you present your talk next 25th at the January event? Thanks!

**Comment by papirfecnike** [comment]
Hi,I’m open to it, yes. What should I know about the event, is it online? Or shall I look for flight tickets and accommodation?Thanks,DoraOn 10 Jan 2024, at 10:16, Yann Torres ***@***.***> wrote:﻿
Hello! Could you present your talk next 25th at the January event? Thanks!

**Comment by onedesert** [comment]
Hello! This month's talk is set, but your submission is queued for next month. We follow a first-come basis and the oldest already confirmed. Stay tuned! 

The events are always onsite at Barcelona.

**Comment by papirfecnike** [comment]
Hello,Great, this means I have more time to prepare more precisely. Please let me know when the next event will take place because I’m flying from Oslo and I need to arrange travel&stay. I really appreciate the opportunity!On 11 Jan 2024, at 11:03, Yann Torres ***@***.***> wrote:﻿
Hello! This month's talk is set, but your submission is queued for next month. We follow a first-come basis and the oldest already confirmed. Stay tuned!
The events are always onsite at Barcelona.

**Comment by onedesert** [comment]
Sure! Base on your response and other outside-BCN speakers last month, we are discussing fixing dates for the next 6 months so we can plan way ahead than a month in advance. I'll keep you posted, as it's not easy with speaker/venue/sponsors involved.

**Comment by papirfecnike** [comment]
Hello,
I guess this won't happen in February, right? 
Thanks,Dora
    On Thursday, January 11, 2024 at 11:24:48 AM GMT+1, Yann Torres ***@***.***> wrote:  
 
 


Sure! Base on your response and other outside-BCN speakers last month, we are discussing fixing dates for the next 6 months so we can plan way ahead than a month in advance. I'll keep you posted, as it's not easy with speaker/venue/sponsors involved.

**Comment by onedesert** [comment]
hello! we have it for this month, we have some dates for the upcoming months but not public yet as they might change.

https://github.com/BarcelonaJS/events/issues/54

**Comment by dmitry-zaets** [comment]
hi @papirfecnike we are planning the event for 30 of May (Thursday), are you available to give a talk? Thanks!

**Comment by papirfecnike** [comment]
> hi @papirfecnike we are planning the event for 30 of May (Thursday), are you available to give a talk? Thanks!

If I remember correctly, I was supposed to give a talk in February. I tried to be flexible, buying plane tickets and making other arrangements, but ultimately, you didn't follow up. Unfortunately, it’s not feasible for me now. Since I'm coming from Oslo, I need to plan and organize everything in advance. I need at least a month's notice.

**Comment by dmitry-zaets** [comment]
Ah, I see. Excuse me, I didn't know that you were flying from another city. In our case, we are currently planning events within a couple of weeks, so it's hard to come up with the date one month in advance.

**Comment by papirfecnike** [comment]
> Ah, I see. Excuse me, I didn't know that you were flying from another city. In our case, we are currently planning events within a couple of weeks, so it's hard to come up with the date one month in advance.

Yes, there was no opportunity to mention this on the form, but I think your event is pretty nice and I wanted to participate. Also, I have an other topic as well, so if you're interested in any time in the future, just let me know. :) Until then, I wish you the best.

**Comment by dmitry-zaets** [comment]
Thank you very much! I've mentioned that in the description, so we take it into account next time!

**Comment by onedesert** [comment]
Hey Dora 👋

We’re planning our next event for April 2025 (likely the last week of the month), and we’d love to include your talk if you’re up for presenting! I hope that is enough time in advance, please let me know.

Thanks!

**Comment by papirfecnike** [comment]
Hello,
what a pleasant surprise! :) I hope you are all doing great.
Unfortunately, I’m presenting on various conferences in April and May, traveling a lot. Maybe in June?...
    On Wednesday, March 26, 2025 at 02:30:54 PM GMT+1, Yann Torres ***@***.***> wrote:  
 
  
Hey Dora 👋

We’re planning our next event for April 2025 (likely the last week of the month), and we’d love to include your talk if you’re up for presenting! I hope that is enough time in advance, please let me know.

Thanks!

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>
onedesert left a comment (BarcelonaJS/talks#6)
Hey Dora 👋

We’re planning our next event for April 2025 (likely the last week of the month), and we’d love to include your talk if you’re up for presenting! I hope that is enough time in advance, please let me know.

Thanks!

**Comment by onedesert** [comment]
Hello! Thanks for your prompt reply, no problem, I'll consider on June again.

**Comment by onedesert** [comment]
Hello! Will you be ready to speak for this month meetup?

**Comment by papirfecnike** [comment]
Hi, I can manage it online. I mentioned several times: I’d need multiple months’ notice ahead to be able to arrange a travel to Barcelona. I’d like to participate your meetup though.
    On Tuesday, September 9, 2025 at 09:40:05 AM GMT+2, Yann Torres ***@***.***> wrote:  
 
 onedesert left a comment (BarcelonaJS/talks#6)
Hello! Will you be ready to speak for this month meetup?

**Comment by onedesert** [comment]
Sorry my bad I didn't reread the entire issue. I'll try to consider but for now we are planning month to month as we do not have stable sponsor. We schedule 2 or max 3 weeks ahead

**Comment by papirfecnike** [comment]
That’s fine. Again, I’m up to an online presentation, if that’s an option. :) Otherwise I wish you folks the best.
    On Wednesday, September 10, 2025 at 11:02:39 AM GMT+2, Yann Torres ***@***.***> wrote:  
 
 onedesert left a comment (BarcelonaJS/talks#6)
Sorry my bad I didn't reread the entire issue. I'll try to consider but for now we are planning month to month as we do not have stable sponsor. We schedule 2 or max 3 weeks ahead
